### DISCUSSION TITLE
how to bulk import users?

### DISCUSSION THREAD

**Discussion by NAIXIL** [requarks/wiki#5158] [Help / Questions] [answered]
I have many users need to be created, except create one by one, any other method can I do?  I notice the bulk button is gray.

**Comment by NAIXIL** [comment]
ps: mysql8, wikijs 2.5.277, plarform:docker

**Comment by NGPixel** [comment, marked as answer]
It's not possible at the moment. This is planned for a future release.

**Comment by Anth0rx** [comment]
Is there progress on this feature? What would be the most efficient way to create multiple users (few 100s) and automatically send out welcome mails with an initial temporary password?

**Comment by EcoSocOrDeath** [comment]
Hey,

I was also wondering if there had been anymore updates to this feature! It would be very helpful for my purposes for wiki.js

Are there any plans to finish it in the near future?

Thanks!

**Comment by P4mes4n** [comment]
The API supports creating new Users. Should be pretty straight forward. The docs even have it as an [example](https://docs.requarks.io/dev/api#create-new-user) and with the playground and ChatGPT even for non-professionals doable.
